pkg/wallet/token.go

```diff
@@ -0,0 +1,60 @@
+package wallet
+
+import (
+	"encoding/json"
+
+	"github.com/nspcc-dev/neo-go/pkg/encoding/address"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// Token represents imported token contract.
+type Token struct {
+	Name     string
+	Hash     util.Uint160
+	Decimals int64
+	Symbol   string
+	Address  string
+}
+
+type tokenAux struct {
+	Name     string       `json:"name"`
+	Hash     util.Uint160 `json:"script_hash"`
+	Decimals int64        `json:"decimals"`
+	Symbol   string       `json:"symbol"`
+}
+
+// NewToken returns new token contract info.
+func NewToken(tokenHash util.Uint160, name, symbol string, decimals int64) *Token {
+	return &Token{
+		Name:     name,
+		Hash:     tokenHash,
+		Decimals: decimals,
+		Symbol:   symbol,
+		Address:  address.Uint160ToString(tokenHash),
+	}
+}
+
+// MarshalJSON implements json.Marshaler interface.
+func (t *Token) MarshalJSON() ([]byte, error) {
+	m := &tokenAux{
+		Name:     t.Name,
+		Hash:     t.Hash.Reverse(), // address should be marshaled in LE but here it is marshaled in LE
```

Comment:
What?

Reply:
`tokenAux` will be marshalled with the default JSON marshaler, which uses `BE`.
Token address is used as LE everywhere so it is better to store it the same way.